src/spec/client/services/product-projections.spec.coffee

```diff
@@ -65,6 +65,18 @@ describe 'ProductProjectionService', ->
   it 'should query for fuzzy', ->
     expect(@service.fuzzy()._queryString()).toBe 'fuzzy=true'
 
+  it 'should query for markMatchingVariants when set to true', ->
+    expect(@service.markMatchingVariants(true)._queryString()).toBe 'markMatchingVariants=true'
+
+  it 'should query for markMatchingVariants when set to false', ->
+    expect(@service.markMatchingVariants(false)._queryString()).toBe 'markMatchingVariants=false'
+
+  it 'should query for not set anything when no value is passed to markMatchingVariants', ->
+    expect(@service.markMatchingVariants()._queryString()).toBe ''
```

Comment:
I did not add any default to the `markMatchingVariants` method, when it's called you need to explicitly pass a boolean to specify whether it should be false or not.